crates/sozo/ops/src/execute.rs

```diff
@@ -6,12 +6,17 @@ use starknet::accounts::{Account, Call};
 use starknet::core::utils::get_selector_from_name;
 
 use super::get_contract_address;
-use crate::commands::execute::ExecuteArgs;
-
-pub async fn execute(args: ExecuteArgs, env_metadata: Option<Environment>) -> Result<()> {
-    let ExecuteArgs { contract, entrypoint, calldata, starknet, world, account, transaction } =
-        args;
 
+pub async fn execute(
+    contract: String,
+    entrypoint: String,
+    calldata: Vec<FieldElement>,
+    starknet: StarknetOptions,
```

Comment:
@glihm Now there are multiple dependencies on this options package that is inside commands/options. Should we move this package to crate?

We can have a call, I will ping you on telegram. Thanks

Reply:
Yes for sure, let's have a call.
But yes, the idea of this PR is to remove those coupling we have here, as they will open the doors for more usage of `ops` of `sozo` more broadly into the project. :+1: